Taskfile.yml

```diff
@@ -393,6 +395,26 @@ tasks:
       - find "config/crd/out" -type f -delete
       - kustomize build config/crd > config/crd/out/azureserviceoperator.yaml
 
+  controller:gen-helm-manifest:
+    desc: Generate helm manifest using Kustomize for the release
+    dir: "v2/"
+    cmds:
+     - sed -i '' 's@\({{.PUBLIC_REGISTRY}}\)\(.*\)@\1{{.CONTROLLER_DOCKER_IMAGE}}@g' ./charts/azure-service-operator/values.yaml
+     - rm -rf charts/azure-service-operator/templates/generated/rm -rf charts/azure-service-operator/templates/generated/
```

Comment:
There seem to be 2 `rm -rf`'s here doing the same thing? I think you can delete one

Reply:
Done!